src/groups/mqb/mqbs/mqbs_filestoreprotocol.h

```diff
@@ -1237,6 +1252,9 @@ struct MessageRecord {
     print(bsl::ostream& stream, int level = 0, int spacesPerLevel = 4) const;
 };
 
+BSLMF_ASSERT(FileStoreProtocol::k_MAX_MSG_REF_COUNT_HARD <=
```

Comment:
We usually put the `BSLMF_ASSERT`s in the `.cpp` file [here](https://github.com/bloomberg/blazingmq/blob/8702c6405ea765f9fb6a25f6db93cf80915899aa/src/groups/mqb/mqbs/mqbs_filestoreprotocol.cpp#L32-L66).